src/components/Resume.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import { createPortal } from 'react-dom';
+import { useNavigate } from 'react-router-dom';
+import { Document, Page } from 'react-pdf/dist/esm/entry.webpack5';
+
+import { cromanResume } from '../../public';
+
+import 'react-pdf/dist/esm/Page/TextLayer.css';
+import 'react-pdf/dist/esm/Page/AnnotationLayer.css';
+
+const ModalContent = ({ onClose }) => {
+  return (
+    <div className="modal">
+      <Document file={cromanResume}>
+        <Page pageNumber={1} />
+      </Document>
+      <button onClick={onClose}>Close</button>
+    </div>
+  );
+};
+
+const Modal = () => {
+  const [showModal, setShowModal] = useState(true);
+  const navigate = useNavigate();
+
+  const handleClose = () => {
+    setShowModal(false);
+    navigate('/');
+  };
+
+  return (
+    <>
+      {showModal &&
+        createPortal(
+          <ModalContent onClose={handleClose} />,
+          document.getElementById('portal')
+        )}
+    </>
+  );
+};
+
+const Resume = () => {
+  return (
```

Comment:
Same here for the automatic return. Also, I appreciate breaking these components out, but the return of `Resume` is so small, I think you could probably make it all one `Resume` component versus three separate components in the same file 🤷